client/state/posts/selectors.js

```diff
@@ -313,7 +314,7 @@ export function getEditedPost( state, siteId, postId ) {
  */
 export function getPostEdits( state, siteId, postId ) {
 	const { edits } = state.posts;
-	return get( edits, [ siteId, postId || '' ], null );
+	return normalizeEditedPost( get( edits, [ siteId, postId || '' ], null ) );
```

Comment:
We may at some point (here or in a subsequent PR) want to memoize this function and/or the normalize function (noted similar follow-up task for post normalization in #6715).